packages/frontend/package.json

```diff
@@ -1,73 +1,70 @@
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@zeit/next-css": "^1.0.1",
-    "autoprefixer": "^9.6.1",
-    "classnames": "^2.2.6",
-    "formik": "^1.5.7",
-    "next": "^9.0.1",
-    "postcss-easy-import": "^3.0.0",
-    "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-dropzone": "^10.1.5",
-    "tailwindcss": "^1.0.4",
-    "yup": "^0.27.0"
-  },
-  "resolutions": {
-    "c21e": "1.1.9"
-  },
-  "lint-staged": {
-    "*.{js,jsx}": [
-      "eslint --ext",
-      "git add"
-    ],
-    "*.{js,jsx,json,css,scss}": [
-      "pretty-quick --staged",
-      "git add"
-    ]
-  },
-  "devDependencies": {
-    "@babel/core": "^7.4.5",
-    "@fullhuman/postcss-purgecss": "^1.2.0",
-    "@storybook/addon-actions": "^5.1.9",
-    "@storybook/addon-knobs": "^5.1.9",
-    "@storybook/addon-links": "^5.1.9",
-    "@storybook/addons": "^5.1.9",
-    "@storybook/react": "^5.1.9",
-    "babel-eslint": "^10.0.2",
-    "babel-loader": "^8.0.6",
-    "c21e": "1.1.9",
-    "cssnano": "^4.1.10",
-    "cucumber": "^5.1.0",
-    "eslint": "5.3.0",
-    "eslint-config-airbnb": "^17.1.1",
-    "eslint-config-prettier": "^6.0.0",
-    "eslint-plugin-import": "2.18.0",
-    "eslint-plugin-jsx-a11y": "6.2.3",
-    "eslint-plugin-prettier": "^3.1.0",
-    "eslint-plugin-react": "7.14.2",
-    "eslint-plugin-react-hooks": "^1.6.1",
-    "gherkin-testcafe": "^2.3.2",
-    "lint-staged": "^8.2.1",
-    "prettier": "^1.18.2",
-    "pretty-quick": "^1.11.1",
-    "testcafe": "^1.3.1"
-  }
+	"name": "ucurtma-frontend",
+	"version": "0.1.0",
+	"description": "Frontend workspace for Ucurtma app",
+	"main": "index.js",
+	"scripts": {
+		"dev": "next",
+		"build": "next build",
+		"start": "next start",
+		"export": "next export",
+		"lint": "lint-staged",
+		"deploy-staging": "aws s3 sync out/ s3://non-prod-ucurtma-app --delete",
+		"storybook": "start-storybook -s ./static -p 6006",
+		"build-storybook": "build-storybook",
+		"e2e-test:chrome:local": "export BASE_URL=http://localhost:3000 && gherkin-testcafe chrome tests/features/**/*.feature tests/step_definitions/**/*.js",
+		"e2e-test:bs": "gherkin-testcafe 'browserstack:chrome' tests/features/**/bs-*.feature tests/step_definitions/**/*.js -S -s tests/sreenshots --skip-js-errors",
+		"e2e-test:firefox": "gherkin-testcafe 'firefox:headless' tests/features/**/*.feature tests/step_definitions/**/*.js -S -s tests/sreenshots --skip-js-errors -r json:tests/reports/report.json"
+	},
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
+		"@zeit/next-css": "^1.0.1",
+		"autoprefixer": "^9.6.1",
+		"classnames": "^2.2.6",
+		"formik": "^1.5.7",
+		"next": "^9.0.1",
+		"postcss-easy-import": "^3.0.0",
+		"prop-types": "^15.7.2",
+		"react": "^16.8.6",
+		"react-dom": "^16.8.6",
+		"react-dropzone": "^10.1.5",
+		"tailwindcss": "^1.0.4",
+		"yup": "^0.27.0"
+	},
+	"resolutions": {
+		"c21e": "1.1.9"
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [ "eslint --ext", "git add" ],
```

Comment:
arrays should break the line :( I started to feel like your ide doesn't respect our prettier config.